planetscale/README.md

```diff
@@ -0,0 +1,40 @@
+# Agent Check: Planetscale
+
+## Overview
+
+PlanetScale can push metrics into Datadog to assist your team with understanding database usage and performance.
+
+## Setup
+
+### Configuration
+
+Follow the steps below to configure your PlanetScale organization to push metrics into Datadog.
+
+1. Create a Datadog API key in your [Datadog Organization Settings][1].
+2. Supply PlanetScale with the Datadog API key in your [PlanetScale Organization Settings][2].
+
+![PlanetScale Organization Settings][3]
+
+## Data Collected
+
+### Metrics
+
+TBD
```

Comment:
I assume this will change before it goes public? Why not put the magic phrase in now? It's always the same: "See [metadata.csv][xxxxxx] for a list of metrics provided by this integration."